<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcedureDialog</class>
 <widget class="QDialog" name="ProcedureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>862</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>862</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ProcedureDialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>541</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отказ</string>
   </property>
  </widget>
  <widget class="QLabel" name="noTeethLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Изберете манипулация!</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Търсене:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectedTeethLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="CommonFields" name="commonFields" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="empty"/>
   <widget class="ObturationView" name="obturWidget"/>
   <widget class="CrownView" name="crownWidget"/>
   <widget class="ImplantView" name="implantWidget"/>
   <widget class="FiberSplintView" name="fiberWidget"/>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>460</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата:</string>
   </property>
  </widget>
  <widget class="DateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
  </widget>
  <zorder>noTeethLabel</zorder>
  <zorder>tableView</zorder>
  <zorder>okButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>label</zorder>
  <zorder>searchEdit</zorder>
  <zorder>selectedTeethLabel</zorder>
  <zorder>commonFields</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>errorLabel</zorder>
  <zorder>label_2</zorder>
  <zorder>dateEdit</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CommonFields</class>
   <extends>QWidget</extends>
   <header>CommonFields/CommonFields.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CrownView</class>
   <extends>QWidget</extends>
   <header>ProcedureFields/CrownView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ObturationView</class>
   <extends>QWidget</extends>
   <header>ProcedureFields/ObturationView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImplantView</class>
   <extends>QWidget</extends>
   <header>View/uiComponents/qt_derived/Widgets/ImplantView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FiberSplintView</class>
   <extends>QWidget</extends>
   <header>ProcedureFields/FiberSplintView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>View/uiComponents/qt_derived/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
